charms/k8s/requirements.txt

```diff
@@ -1,2 +1,4 @@
 ops >= 2.2.0
 pydantic == 1.*
+charm-lib-reconciler @ git+https://github.com/charmed-kubernetes/charm-lib-reconciler
+charm-lib-contextual-status @ git+https://github.com/charmed-kubernetes/charm-lib-contextual-status@feat/custom-exit-status
```

Comment:
should we get this merged first?

Reply:
I think we need to explore the actual improvements we want to implement in the libraries. So, I'll point to the main branches.